runtime/android/core/src/org/xwalk/core/XWalkCoreWrapper.java

```diff
@@ -29,6 +31,9 @@
     private static final String WRAPPER_PACKAGE = "org.xwalk.core";
     private static final String BRIDGE_PACKAGE = "org.xwalk.core.internal";
     private static final String TAG = "XWalkLib";
+    private static final String XWALK_CORE_LIB_DIR = "extracted_xwalkcore";
```

Comment:
This name is ambiguous. Please use something like "XWALK_CORE_EXTRACTED_DIR" to indicate it's extracted explicitly